packages/strapi-admin/controllers/Admin.js

```diff
@@ -43,6 +43,14 @@ module.exports = {
     }
   },
 
+  plugins: async ctx => {
+    try {
+      ctx.send({ plugins: strapi.plugins });
```

Comment:
Can you expose just informatios you need. Not all `strapi.plugins` object.

Reply:
Should be ok now